File: LinkedLists/JAVA/LinkedList/src/SingleLinkedList/Node.java
/**
 * 
 */
package SingleLinkedList;

/**
 * @author Sakshi Panday Jun 21, 2018
 */
public class Node {

	Node next = null;
	String data;

	public Node(String value) {
		setData(value);
	}

	public Node getNext() {
		return next;
	}

	public void setNext(Node next) {
		this.next = next;
	}

	public String getData() {
		return data;
	}

	public void setData(String data) {
		this.data = data;
	}

}

File: LinkedLists/Python 3/SingleLinkedList/Node.py
# -*- coding: utf-8 -*-
"""
Created on Sat Jun 23 08:39:59 2018

@author: Sakshi Panday

@Overview This is used to define Node for Single linked List
"""

class node(object):
    
    def __init__(self, data):
        self.nextLink = None
        self.data = data
        
    def getNext(self):
        return self.nextLink
    
    def getData(self):
        return self.data
    
    def setNext(self, newNextLink):
        self.nextLink = newNextLink
        
    def setData(self, newData):
        self.data = newData